doc/material-getting-started.md

```diff
@@ -260,6 +260,22 @@ If you would like Uno.Material to use a different font, you can override the def
                    FontOverrideSource="ms-appx:///PROJECT_NAME/Styles/Application/MaterialFontsOverride.xaml" />
     ```
 
+## Using C# Markup
+
+Uno Material also has support for C# Markup through a [Uno.Material.WinUI.Markup](https://www.nuget.org/packages/Uno.Material.WinUI.Markup) NuGet Package. 
+
+To get started with Uno Material in your C# Markup application, install the `Uno.Material.WinUI.Markup` NuGet package in your **App Code Library** project and your platform heads and add the following code to your `AppResources.cs`:
```

Comment:
```suggestion
To get started with Uno Material in your C# Markup application, add the `Uno.Material.WinUI.Markup` NuGet package to your **App Code Library** project and your platform heads.
Then, add the following code to your `AppResources.cs`:
```

Reply:
also clarify whats "App Code Library"?

Reply:
App Code Library is explained at line 37